pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -49,6 +49,9 @@ var (
 )
 
 func init() {
+	if _, runOperator := os.LookupEnv("RUN_OPERATOR"); runOperator {
```

Comment:
Can we encapsulate this in a fuction like `isOperator()` we have multiple places.